--- a/bfd/configure	2018-01-27 14:58:29.000000000 +0000
+++ b/bfd/configure	2026-07-09 10:22:49.351109552 +0000
@@ -14379,6 +14379,7 @@
     hppa_som_vec)		 tb="$tb som.lo" ;;
     i370_elf32_vec)		 tb="$tb elf32-i370.lo elf32.lo $elf" ;;
     i386_aout_vec)		 tb="$tb i386aout.lo aout32.lo" ;;
+    i386_os2_vec)		 tb="$tb i386os2.lo aout32.lo" ;;
     i386_aout_bsd_vec)		 tb="$tb i386bsd.lo aout32.lo" ;;
     i386_aout_dynix_vec)	 tb="$tb i386dynix.lo aout32.lo" ;;
     i386_aout_fbsd_vec)		 tb="$tb i386freebsd.lo aout32.lo" ;;
