--- a/bfd/targets.c	2018-01-13 13:31:15.000000000 +0000
+++ b/bfd/targets.c	2026-07-09 10:22:30.361093152 +0000
@@ -665,6 +665,7 @@
 extern const bfd_target hppa_som_vec;
 extern const bfd_target i370_elf32_vec;
 extern const bfd_target i386_aout_vec;
+extern const bfd_target i386_os2_vec;
 extern const bfd_target i386_aout_bsd_vec;
 extern const bfd_target i386_aout_dynix_vec;
 extern const bfd_target i386_aout_fbsd_vec;
@@ -1096,6 +1097,7 @@
 	&i370_elf32_vec,
 
 	&i386_aout_vec,
+	&i386_os2_vec,
 	&i386_aout_bsd_vec,
 #if 0
 	&i386_aout_dynix_vec,
